node/resolvers/profile/services.ts

```diff
@@ -68,7 +68,7 @@ export async function getPayments(context: Context) {
 
   const paymentsRawData = await profile.getUserPayments(currentProfile)
 
-  if (!paymentsRawData) {
+  if (!paymentsRawData || paymentsRawData.availableAccounts == null) {
```

Comment:
Not sure if this is the best check to do... What if this is undefined? Maybe, this is a better approach:

```suggestion
  if (!paymentsRawData?.availableAccounts) {
```